modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/CompareRuns/index.tsx

```diff
@@ -24,6 +24,8 @@ import {TestrayRun} from '../../services/rest';
 const CompareRunDetails: React.FC<{runs: TestrayRun[]}> = ({runs = []}) => {
 	const {colors, columns, data} = useTableChartData();
 
+	document.title = i18n.translate('compare-cases');
+
```

Comment:
I think you could use, sub
```suggestion
	document.title = i18n.sub('compare-x', 'cases');

```